packages/calico/extra/dcos-calico-felix.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=DC/OS Calico Node (felix): Calico Core component responsible for programming routes and security policy
+
+[Service]
+EnvironmentFile=/opt/mesosphere/environment
+EnvironmentFile=/opt/mesosphere/etc/calico/calico-node.env
+EnvironmentFile=/opt/mesosphere/etc/calico/calico-node-datastore.env
+ExecStartPre=/opt/mesosphere/bin/bootstrap dcos-calico-felix
+ExecStart=/opt/mesosphere/bin/start-calico-felix.sh
```

Comment:
By wrapping all the steps inside a script, we can also obtain the failed steps, and it helps me better maintain the code with comments in the script.